search-service/src/test/kotlin/com/egm/stellio/search/service/AttributeInstanceServiceTests.kt

```diff
@@ -365,6 +365,64 @@ class AttributeInstanceServiceTests : WithTimescaleContainer, WithKafkaContainer
             }
     }
 
+    @Test
+    fun `it should only return the last n instances without aggregated or temporal values specified`() = runTest {
```

Comment:
I don't really see the difference between this test and the one just above (`it should only return the last n instances asked in the temporal query`)

Reply:
here I removed` withAggregatedValues` and `aggrPeriodDuration`

Reply:
in the test above, `withAggregatedValues` is also not provided (and so is false), nor `aggrPeriodDuration`

Reply:
an `aggrMethods` was added. But since you said it should also be removed then this test is unecessary

Reply:
if `withAggregatedValues` is false, `aggrMethods` does not make sense.